ui/apps/platform/scripts/cypress-ocp.sh

```diff
@@ -28,8 +29,8 @@ fi
 artifacts_dir="${TEST_RESULTS_OUTPUT_DIR:-cypress/test-results}/ocp-artifacts"
```

Comment:
This change from my branch isn't quite right either - it looks like the video artifacts are not available in the job results. This is something I can fix separately though. I was just looking for video evidence of the test working beyond the text output.